internal/generator/fluentd/ingress.go

```diff
@@ -207,43 +201,14 @@ const KubernetesMetadataPlugin string = `
 const ParseJsonFields string = `
 {{define "parseJsonFields" -}}
 # {{.Desc}}
-<filter kubernetes.journal.**>
-  @type parse_json_field
-  merge_json_log 'false'
-  preserve_json_log 'true'
-  json_fields 'log,MESSAGE'
-</filter>
-
-<filter kubernetes.var.log.pods.**>
-  @type parse_json_field
-  merge_json_log 'false'
-  preserve_json_log 'true'
-  json_fields 'log,MESSAGE'
-</filter>
-
 <filter kubernetes.var.log.pods.**_eventrouter-**>
   @type parse_json_field
   merge_json_log true
   preserve_json_log true
-  json_fields 'log,MESSAGE'
-</filter>
-{{end}}
-`
-
-const CleanKibanaLogs string = `
-{{define "cleanKibanaLogs" -}}
-# {{.Desc}}
-<filter **kibana**>
-  @type record_transformer
-  enable_ruby
-  <record>
-    log ${record['err'] || record['msg'] || record['MESSAGE'] || record['log']}
-  </record>
-  remove_keys req,res,msg,name,level,v,pid,err
+  json_fields 'message'
```

Comment:
previously was 'MESSAGE' in upper case

Reply:
this comes from 3.11 where we were pulling container logs using the journal driver.  It is no longer relevant since we only pull container logs and I have changed the group capture parameters: https://github.com/openshift/cluster-logging-operator/pull/1563/files#diff-cdd5a7217e8d3eeb415a8f7f720e09303a82c2f9225ee7484d8bdabc11901f1eR148